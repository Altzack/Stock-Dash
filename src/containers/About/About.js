import React from 'react';
import styled from 'styled-components/macro';
import { useResponsive } from '../common/responsiveComponents';

const ContentContainer = styled.div`
  flex-direction: column;
  font-weight: 200;
  display: flex;
  align-items: center;
  margin: auto;
  @media (max-width: 480px) {
    font-size: 20px;
    padding: 10px;
  }

  @media (min-width: 750px) {
    font-size: 20px;
    max-width: 650px;
    padding: 10px;
  }

  @media (min-width: 1023px) {
    max-width: 900px;
    font-size: 30px;
    padding: 10px;
  }
  @media (min-width: 1023px) {
    max-width: 900px;
    font-size: 30px;
    padding: 10px;
  }
`;

const PageContainer = styled.div`
  padding-top: 42px;
  font-family: Rubik;
  padding: 15px;
  display: flex;
  font-size: 28px;
  color: #fff;
  align-items: center;
  width: 100%;
  flex-direction: column;
`;

const Title = styled.h1`
  font-size: 50px;
  font-weight: 300;
  text-align: left;
  letter-spacing: 3px;
  text-transform: capitalize;
  color: #fff;
  margin-bottom: 10px;
  @media (max-width: 480px) {
    font-size: 30px;
  }
`;

export default function LandingPage() {
  const { isTabletOrMobile } = useResponsive();
  return (
    <PageContainer mobile={isTabletOrMobile}>
      <Title>About Stock-Dash</Title>
      <br />
      <ContentContainer>
        Stock-Dash was built as my second full stack capstone for Bloc.
        <br />
        <br />
        The idea for Stock-Dash came from my interest in trading stocks. The
        dashboard allows you to get the latest news and pricing for a symbol, as
        well as see the past month's price data visualized. With the watchlist
        feature you can easily navigate to your top 5 and keep an eye on your
        favorite tickers.
        <br />
        <br />
      </ContentContainer>
    </PageContainer>
  );
}
