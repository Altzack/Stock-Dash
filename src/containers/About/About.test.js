import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import About from './About';

it('renders a About by default', () => {
  const wrapper = shallow(<About />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
