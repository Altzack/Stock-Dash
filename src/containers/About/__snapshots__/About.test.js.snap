// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a About by default 1`] = `
<div>
  about
</div>
`;
